go/modelzooserver/modelzooserver.go

```diff
@@ -413,15 +424,21 @@ func (s *modelZooServer) ReleaseModel(ctx context.Context, req *pb.ReleaseModelR
 	defer sqlf.Close()
 
 	buf := make([]byte, 4096)
-	for {
-		n, e := sendFile.Read(buf)
-		if e == io.EOF {
-			break
-		}
-		_, err = sqlf.Write(buf[:n])
-		if err != nil {
+	eof := false
+	for !eof {
+		n, err := sendFile.Read(buf)
+		if err == io.EOF {
+			eof = true
```

Comment:
``` golang
if err == io.EOF {
    break
}
```
that we can remove the local boolean variable `eof`.

Reply:
No. I found that when `err == io.EOF`, `n` may be larger than 0 when using our `sqlfs` package.